WEEK ONE/Challenge 1, Even Or Odd.py

```diff
@@ -0,0 +1,13 @@
+print("Welcome to the number identificator. Enjoy and have fun!")
+
+fav_num = int(input("Type your favourite number: "))
+
+if fav_num % 2 == 0:
+    print('Your number favourite is an even number')
+
+else:
+    print('Your number favourite is an odd number')
+
+print("Thank you for playing the number identificator. Hope you had fun! "
+      "\n"
+      "Please give comment or feedback: ")

```

Comment:
What happens when I type something like `Nice` as my comment?

Reply:
The last line looks like it expects an input from the user, but there is no `input` function

Reply:
Noted. 

I will add the input function for the feedback, and also an acknowledgement of feedback

Reply:
Code Updated!